cmake/sanitize.cmake

```diff
@@ -77,7 +77,7 @@ if (SANITIZE)
         endif()
 
     elseif (SANITIZE STREQUAL "undefined")
-        set (UBSAN_FLAGS "-fsanitize=undefined -fno-sanitize-recover=all -fno-sanitize=float-divide-by-zero")
+        set (UBSAN_FLAGS "-fsanitize=undefined -fno-sanitize-recover=all -fno-sanitize=float-divide-by-zero -fno-sanitize=unsigned-integer-overflow")
```

Comment:
Need to provide comments, like
- it is set implicitly by OSSFuzz (?);
- it is not undefined behaviour according to C++ standard, compilers or CPU;
- it is actively used in hash functions and random number generators.